src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea(): number,
 }
 
-export class Triangle {
+export function checkTriangle(sides:number[]):boolean {
+  const maxSide = Math.max(...sides);
+  let sum = 0;
+
+  sides.forEach((el:number) => {
+    if (el !== maxSide) {
+      sum += el;
+    }
+  });
 
+  return sum >= maxSide;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  constructor(
+    public shape: 'triangle',
```

Comment:
the constructor doesn't receive the shape. it should be set as 'triangle' for all instances. check it everywhere